### DISCUSSION TITLE
Hide YouTube endscreen video recommendations

### DISCUSSION THREAD

**Discussion by shbach** [jordwest/news-feed-eradicator#375] [Ideas]
The new YouTube support is great but it's just missing one key element which is the video recommendations shown at the end of videos. These algorithmically generated recommendations are pretty much a continuation of the newsfeed so it's very much in the spirit of this extension to hide it.

![image](https://user-images.githubusercontent.com/5298745/107322509-74d9da80-6a59-11eb-8f2f-c51b86f1bf1a.png)

With this feature added I'll be able to consolidate and no longer need to use extensions such as [DF YouTube](https://addons.mozilla.org/en-US/firefox/addon/df-youtube/) which doesn't seem to be actively maintained anymore.

**Comment by Romnoel** [comment]
Agreed. 

Respectable reccomendations by the way. I like a bit of Mark Rober and Minecraft lol. Not the best for productivity though. Bit of a rabbithole.
